src/help.html

```diff
@@ -4,83 +4,106 @@
 description: Have questions about the BeakBox? We've got answers! Find tutorials, how-to guides & tips and tricks to make the most out of your parrot toy, Beakbox.
 header-title: Help Centre
 header-paragraph: Everything you need to know about BeakBox. Can't find the answer you're looking for? Please <a href="mailto:contact@mybeakbox.com">chat</a> to our friendly team.
-image: /imgs/beakbox-transparent.png
+image: /imgs/beakbox-front-v2-transparent.png
 ---
 
 <section>
-  <article class="popular-questions light-background main-container layout-grid">
+  <div class="main-container main-container--narrow">
+    <h2 class="colored-text">Quick links</h2>
+    <ul>
+      <li><a href="#popular-questions">Frequently asked questions</a></li>
+      <li><a href="#shipping-and-delivery">Shipping and delivery</a></li>
+      <li><a href="#getting-started">Getting started</a></li>
+      <li><a href="#beakbox-web-app">BeakBox web app</a></li>
+      <li><a href="#beakbox-troubleshooting">BeakBox troubleshooting</a></li>
+    </ul>
+  </div>
+</section>
+
+<section>
+  <article class="popular-questions light-background main-container main-container--narrow">
     <div class="flex-container">
-      <h2 id="popular-questions">Frequently Asked Questions</h2>
+      <h2 class="colored-text" id="popular-questions" >Frequently asked questions</h2>
```

Comment:
The extra space in the opening tag may mean your code formatter is not working as it should. Let's look in to it when we're both in the office.